docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_Coverflow.htm

```diff
@@ -0,0 +1,231 @@
+        &lt;li&gt;&lt;img src=&quot;sample3.png&quot;&gt;&lt;/li&gt;
+    &lt;/ul&gt;
+</pre>
+
+</li>
+
+</ul>
+
+<h2><a id="options-for widget-coverflow-wearable"></a>Options</h2>
+
+<p>The options for a component can be defined as <code>data-effect</code> attributes or passed as parameters to the constructor.</p>
+
+<p>You can change an option for the component using the <code>option</code> method.</p>
+
+	<h3>Summary</h3>
+
+	<table>
+		<tbody>
+		<tr>
+			<th>Option</th>
+			<th>Input type</th>
+			<th>Default value</th>
+			<th>Description</th>
+		</tr>
+
+
+			<tr>
+				<td class="option"><span style="font-family: Courier New,Courier,monospace">data-effect</span></td>
+				<td class="option">string | null</td>
+				<td class="option">"coverflow"</td>
+				<td class="option">Effect type.</td>
+			</tr>
+
+		</tbody>
+	</table>
+
+
+
+<h2><a id="methods-coverflow-wearable"></a>Methods</h2>
+
+<p>To call a method on the component, use one of the existing APIs:</p>
+
+<pre class="prettyprint">
+
+&lt;script type=&quot;text/javascript&quot; src=&quot;lib/tau/libs/tau.coverflow.js&quot;&gt;&lt;/script&gt;
+&lt;script type=&quot;text/javascript&quot; src=&quot;lib/tau/libs/tau.js&quot;&gt;&lt;/script&gt;<br>
+&lt;script&gt;
+    var coverflowElement = document.getElementsByClassName(&quot;ui-coverflow&quot;),
+        coverflow = tau.widget.CoverFlow(coverflowElement);
+
+    coverflow.value(&quot;carousel&quot;);
+&lt;/script&gt;
+</pre>
+
+	<h3>Summary</h3>
+	<table class="informaltable">
+		<thead>
+		<tr>
+			<th>Method</th>
+			<th>Description</th>
+		</tr>
+		</thead>
+		<tbody>
+
+
+				<tr>
+					<td>
+						<pre class="intable prettyprint">string <a href="#method-value">value</a>(string? value) </pre>
+					</td>
+					<td><p>Gets or sets a value.</p></td>
+				</tr>
+
+
+		</tbody>
+	</table>
+	<dl>
+
+
+
+				<dt class="method" id="addidp28072"><code><b><span class="methodName"
+																   id="method-value">value</span></b></code></dt>
+				<dd>
+					<div class="brief">
+						<p>Gets or sets a value.</p>
+					</div>
+					<div class="synopsis">
+						<pre class="signature prettyprint">string value(string? value) </pre>
+					</div>
+
+						<p><span class="version">Since: </span>
+							5.5
+						</p>
+
+					<div class="description">
+						<p>
+							<p>The method returns the inner text of the coverflow effect or sets the text of the coverflow effect.</p>
+
+
+						</p>
+					</div>
+
+					<div class="parameters">
+						<p><span class="param">Parameters:</span></p>
+					<table>
+					<tbody>
+					<tr>
+						<th>Parameter</th>
+						<th>Type</th>
+						<th>Required/optional</th>
+						<th>Default value</th>
+						<th>Description</th>
+					</tr>
+
+
+					<tr>
+						<td><span style="font-family: Courier New,Courier,monospace">value</span></td>
+						<td>string</td>
+						<td>Optional</td>
+						<td>"coverflow"</td>
+						<td>Text to set for the coverflow effect.<br>
```

Comment:
"Text to set the coverflow effect." looks correct.

Reply:
ok. fixed.

done.